### DISCUSSION TITLE
making software available for installation with brew - what's the process?

### DISCUSSION THREAD

**Discussion by Karl-Bee** [Homebrew/discussions#4585] [Writing Formulae/Casks]
### Output of `brew config`

```shell
not required
```

### Output of `brew doctor`

```shell
not required
```

### Description of issue

Dear all,

I would love to be able to install https://www.audiocupcake.com/ via brew.
what are the steps?
I assume this would be a cask, however sorry, I don't know for sure, maybe formula etc.

can anyone make it available if there's a permanent download link?
Does the developer need to take action?
(I'm a user only, should I then contact the developer? - or more common: Whats the process behind adding new software?)

Thank you,
Best Regards,

Karl

sorry for the question, but all search terms i tried didn't show up great results.

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/149564

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew
